cmd/minishift/cmd/ip.go

```diff
@@ -57,7 +57,11 @@ var ipCmd = &cobra.Command{
 		if configureAsDynamic {
 			minishiftNetwork.ConfigureDynamicAssignment(host.Driver)
 		} else if configureAsStatic {
-			minishiftNetwork.ConfigureStaticAssignment(host.Driver)
+			msg, err := minishiftNetwork.ConfigureStaticAssignment(host.Driver)
+			if err != nil {
+				atexit.ExitWithMessage(1, err.Error())
+			}
+			fmt.Println(msg)
```

Comment:
@praveenkumar another Println statement, can you check if it is required?

Reply:
@LalatenduMohanty yes this is required.